pkg/client/config.go

```diff
@@ -0,0 +1,15 @@
+package client
+
+import (
+	configtime "github.com/keep-network/keep-ecdsa/internal/config/time"
+)
+
+// Config contains configuration for tss protocol execution.
+type Config struct {
+	// Period to check keeps for awaiting key generation on client start.
+	AwaitingKeyGenerationLookback configtime.Duration
+
+	// Timeout for key generation and signature calculation.
+	KeyGenerationTimeout configtime.Duration
+	SigningTimeout       configtime.Duration
+}
```

Comment:
What do you think about defining three `Get*` functions in `Config` and handling there the default/non-default values? We can also define the defaults in this file.